### ISSUE TITLE
Run on alternate dekstop

### ISSUE THREAD

**Issue by Bodup** [keeweb/keeweb#1015]
**TL;DR**
Run KeeWeb on a different Desktop (see [here](https://docs.microsoft.com/en-us/windows/desktop/winstation/window-station-and-desktop-creation)) for GUI isolation.

**Longer version**
Chrome and Firefox run their processes in desktops other than the default Winsta0\Default (e.g Service-0x0-105f127$\sbox_alternate_desktop_0x3194).
This prevents them from hooking into other Windows, in case the process is compromised by an attacker.

KeePass allows the user to enter his/her master password on the "secure desktop" (It's not actually the same desktop that UAC uses, instead it makes it look as if it is by setting a dimmed screenshot as it's background image). This prevents other programs from reading what's being typed in the master password prompt.

**My suggestion**
My suggestion would be for KeeWeb to run on a different desktop other than Winsta0\Default (without this being visible to the user, just like with Chrome and Firefox). This isolates its GUI from other applications. 

(I personally run KeeWeb as an administrator because its higher integrity level prevents other programs from interacting with it. e.g reading it's memory contents. But that doesn't really help if it's running on the same desktop; any application can hook into its GUI and read what's inside.)

**Comment by antelle** [comment]
Not sure if it's possible in Electron, but it would be nice to have of course.
